openapi/components/paths/system.yaml

```diff
@@ -27,6 +27,69 @@ paths:
       x-badges:
         - color: gold
           label: 'Status: Early Access (Still Under Development)'
+  /jams:
+    get:
+      summary: Show jams list
+      tags:
+        - system
```

Comment:
Is it appropriate to consider this as part of the systems grouping?

Reply:
I'm unsure, I put it here due to getinfopush also being here. Would you rather I made a new grouping just for jams?

Reply:
I think that would be preferred, but I'm not against putting it in system. I worry that system is becoming a "miscellaneous" category, and would like to prevent that.

Reply:
All right, will do that then